src/main/webapp/index-script.js

```diff
@@ -37,4 +37,27 @@ function loadMainButtons() {
   logInButton.addEventListener("click", () => {
     return; // TO-DO: Fetch log-in info here.
   });
+}
+
+// Search using text query
+function searchByText(){
+  // Create the places service.
+  const service = new google.maps.places.PlacesService(map);
+  
+  // Perform a query (hard-coded to be the Googleplex for right now)
+  var request = {
+      query: 'Googleplex',
+      fields: ['place_id', 'geometry']
+  }
+
+  service.findPlaceFromQuery(request, function(results, status) {
+    if (status === google.maps.places.PlacesServiceStatus.OK) {
+      for (var i = 0; i < results.length; i++){
+          // Populate the screen
```

Comment:
Great job Claudia! I think we should discuss how we want to handle the results from searchByText() and searchByMap(). I think  we should have a function that basically just accepts the results and handles them separately.

Reply:
I agree - let's talk more about this before/after our pod sync?